tsconfig.json

```diff
@@ -6,7 +6,8 @@
 			"~/*": ["./src/*"]
 		},
 		"jsx": "preserve",
-		"jsxImportSource": "solid-js"
+		"jsxImportSource": "solid-js",
+		"resolveJsonModule": true
```

Comment:
Enabled this to get proper types for the imported `authors.json` in `src/content/config.ts`

Reply:
our base config already includes this: https://github.com/withastro/astro/blob/735767f8dfe302834e48c2383c387b0e41c2f006/packages/astro/tsconfigs/base.json#L11-L12

Reply:
Hmm, I could swear I changed something, but maybe I actually fixed it by switching from an array to an object.